### ISSUE TITLE
Error resolving `preact-router/match`

### ISSUE THREAD

**Issue by t47io** [developit/preact-router#181]
Following the README, but encountered this error with version `2.5.1`:
```
Module not found: Error: Can't resolve 'preact-router/match' in ...
```
I'm using webpack 2.4.1. I found `preact-router` points to `./node_modules/preact-router/dist/preact-router.es.js`, in which there is no `match` or `Match`.

========
Also, is there a difference between the `Link` component in `preact-router` vs. `preact-router/match`?

(My understanding is that `Link` inside `preact-router/match` is `Match` treated version of the one in `preact-router`)

**Comment by t47io** [comment]
I'm now using
```
import { Link } from 'preact-router';
```

This does successfully  route pages and push history. However,  the `activeClassName` does not seem working.

**Comment by t47io** [comment]
Reading the source code, it seems `src/match.js` is not imported anywhere thus not included in the built js bundle?? @developit

**Comment by developit** [comment]
Your initial understanding is correct. I forgot to whitelist `match.js` when publishing the package, so its missing. Will fix asap!

**Comment by t47io** [comment]
Now works perfectly in `2.5.2`.

This issue is resolved by commit 723294b9ffc92cc51cca572321611aa3b18fbadd, please close.
